ooniprobe/Model/Database/Measurement.mm

```diff
@@ -96,4 +96,48 @@ -(void)deleteObject{
     [self remove];
 }
 
+-(void)getExplorerUrl:(void (^)(NSString*))successcb onError:(void (^)(NSError*))errorcb {
+    NSURLComponents *components = [[NSURLComponents alloc] init];
+    components.scheme = @"https";
+    components.host = @"api.ooni.io";
+    components.path = @"/api/v1/measurements";
+    NSURLQueryItem *reportIdItem = [NSURLQueryItem
+                                    queryItemWithName:@"report_id"
+                                    value:self.report_id];
+
+    if ([self.test_name isEqualToString:@"web_connectivity"]){
```

Comment:
Maybe here a comment should explain that `web_connectivity` is the only test using `input` for now?